.gitignore

```diff
@@ -1,6 +1,9 @@
 node_modules/
 cache/
 out/
-broadcast/
```

Comment:
Why remove this?

Reply:
@gregfromstl I did not remove it:  https://github.com/nation3/court/pull/31/commits/46d49c060e3684355899bd335cb71438af8fc5bd#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947

I think that deletion happened when you merged the main branch.  Was there a merge conflict when you did the update?  https://github.com/nation3/court/commit/a738b1bea74b4de993bd10c3f83c496f52de9f84